src/segment-loader.js

```diff
@@ -992,7 +992,7 @@ export default class SegmentLoader extends videojs.EventTarget {
       this.sourceUpdater_.removeAudio(start, end, removeFinished);
     }
 
-    if (this.loaderType_ === 'main' && this.startingMedia_.hasVideo) {
+    if (this.loaderType_ === 'main' && this.startingMedia_ && this.startingMedia_.hasVideo) {
```

Comment:
protect against undefined `this.startingMedia_` as that is the state it starts in.